pluto/src/input/Switch.css

```diff
@@ -62,9 +62,26 @@
     &.pluto--disabled {
         .pluto-input-switch__slider {
             background: var(--pluto-gray-l3);
+            cursor: not-allowed;
         }
         .pluto-input-switch__slider::before {
             background: var(--pluto-gray-l4);
         }
+
+        .pluto-input-switch__input:checked + .pluto-input-switch__slider {
```

Comment:
Do we need this? Not sure what this does, and I don't see any different looking switches